front/templates/front/raw.html

```diff
@@ -39,7 +39,7 @@ <h1>Bostar Deck</h1>
       <!-- unauthenticated raw.html-->
       <div class="contentttt class" style>
              <p>Nie jesteś zalogowany!</p>
-             <a href="login" class="btn btn-w-m btn-success">Zaloguj się !</a>
+             <a href="accounts/login" class="btn btn-w-m btn-success">Zaloguj się !</a>
```

Comment:
Use django reverse url mechanism and don't hard code urls in templates

Reply:
`django reverse url mechanism`

What do you mean?

I am not used this before - can you have sent me link on web where they use this mechanism?

Reply:
done